workspaces/snapshot-tests/outputs/fe00636b02f665bf482575684386a6d1a6fca72da0ed9f6a5b8c8a71a88f0a97/af5c6563210c4c3efd24538b6de1d243bbc1ed32a9d107caddb480d2b6efccb3/output.snapshot.txt

```diff
@@ -17,10 +17,38 @@
           ]
         },
         "requestsTrail": {
-          "SpecRoot": {}
+          "SpecPath": {
+            "pathId": "root"
+          }
         }
       }
     },
-    "6005d492e893247f"
+    "4822093850025955565"
+  ],
+  [
+    {
+      "UnmatchedResponseBodyContentType": {
```

Comment:
not sure how to interpret this one...

Reply:
That is.. odd? Why would any of that change because of the way fingerprints work? Perhaps a latent snapshot from some other PR? I haven't seen any failing PR merges though 🤔